evm/vm_state.py

```diff
@@ -1,3 +1,7 @@
+from typing import (  # noqa: F401
+    Type
+)
+
```

Comment:
There should be no blank lines between stdlib imports, and they should be ordered alphabetically